test/e2e/basic_test.go

```diff
@@ -505,6 +513,7 @@ func getRoundNumber(data *TestData, podName string) (uint64, error) {
 // the previous "round" which are no longer needed (e.g. in case of changes to the cluster / to
 // Network Policies) are removed correctly.
 func TestDeletePreviousRoundFlowsOnStartup(t *testing.T) {
+	skipIfRunCoverage(t)
```

Comment:
`skipIfRunCoverage` should take a `reason` string as parameter and log it as well (e.g. see `skipIfProviderIs`), which should use to explain why the test is being skipped

Reply:
sounds good, added reason.